memory/test/test_memory.py

```diff
@@ -0,0 +1,42 @@
+import concurrent
+import unittest
+import uuid
+from memory.client import BlockingMemoryClient
+
+from hailtop.config import get_user_config
+from hailtop.google_storage import GCS
+
+
+class FileTests(unittest.TestCase):
+    def setUp(self):
+        bucket_name = get_user_config().get('batch', 'bucket')
+        token = uuid.uuid4()
+        self.test_path = f'gs://{ bucket_name }/memory-tests/{ token }'
+
+        self.fs = GCS(concurrent.futures.ThreadPoolExecutor(), project='hail-vdc')
```

Comment:
You need to pass the project in, it is a CI global parameter.  See batch deployment, for example.

I'm not totally sure why GCS needs a project.  Storage is globally scoped, and you only need a project for requestor pays.  Our aiogoogle client should fix this.